src/scenedit/scen.core.cpp

```diff
@@ -3139,17 +3142,24 @@ void edit_scenario_events() {
 	using namespace std::placeholders;
 	
 	cDialog evt_dlg(*ResMgr::dialogs.get("edit-scenario-events"));
+	cStack& stk = dynamic_cast<cStack&>(evt_dlg["list"]);
+	evt_dlg["prev"].attachClickHandler(std::bind([&] { return stk.setPage(0); }));
+	evt_dlg["next"].attachClickHandler(std::bind([&] { return stk.setPage(1); }));
```

Comment:
There's no reason to use `std::bind` on a lambda like this. Just use the lambda. (Also, `[&stk]` please.)